web/src/components/KebabMenu.tsx

```diff
@@ -80,8 +80,6 @@ const Content = styled.div`
 const ToggleButton = styled.button`
   background-color: transparent;
   border: none;
-  /* TODO IGAPP-889 remove this margin after implementing new search icon to fix wrong viewBox of svg */
-  margin-top: 6px;
```

Comment:
I am sorry for the confusion but actually this is still needed for now :/ However, I am already working on it in #1882, so could you please readd the margin and just change the issue number to #1882? Thanks!